initial-promises/node-server/inventory/1.0/fusionAgent.cf

```diff
@@ -27,6 +27,17 @@
 
 bundle agent doInventory
 {
+  classes:
+    "disable_inventory" expression => fileexists("${g.rudder_disable_inventory_file}");
+  methods:
+    ignore_disable_inventory|!disable_inventory::
+      "any" usebundle => doInventory_always;
+  reports:
+    !(ignore_disable_inventory|!disable_inventory)::
+      "INFO: The file ${g.rudder_disable_inventory_file} is present. Rudder will not send an inventory.";
```

Comment:
Can you add "Run 'rudder agent inventory --force' to override this temporarily." ?